media/js/base/global.js

```diff
@@ -114,10 +114,11 @@ function isFirefoxUpToDate(latest) {
     return (latestFirefoxVersion <= fx_version);
 }
 
-// Used on the plugincheck page to also support SeaMonkey
-function isSeaMonkey(userAgent) {
+// Used on the plugincheck page to also support all browsers based on Gecko.
+// Not including Camino as it has reached EOL [http://caminobrowser.org/]
+function isLikeFirefox(userAgent) {
     var ua = userAgent || navigator.userAgent;
-    return /SeaMonkey/i.test(ua);
+    return (/Iceweasel/i).test(ua) || (/SeaMonkey/i).test(ua);
```

Comment:
Just thinking about general reuse since this is in global.js. Part of me expects a function `isLikeFirefox` to check for `like Firefox`. Seeing as Camino is EOL I'm fine not including it here, though maybe something more specific would be good?